packages/create-waku/src/index.ts

```diff
@@ -98,17 +101,20 @@ async function doPrompts() {
   const templateNames = await getTemplateNames(templateRoot);
 
   const defaultProjectName = 'waku-project';
-  let targetDir = '';
+  let targetDir = values['project-name'] || '';
 
   try {
     const result = await prompts(
       [
         {
           name: 'projectName',
-          type: 'text',
+          type: values['project-name'] ? null : 'text',
           message: 'Project Name',
           initial: defaultProjectName,
-          onState: (state: any) => (targetDir = String(state.value).trim()),
+          onState: (state: any) =>
+            (targetDir =
+              String(values['project-name']).trim() ||
```

Comment:
This might be unnecessary... checking...

Reply:
Yeah, removed the change from there.

Reply:
Do we still need this?
```js
let targetDir = values['project-name'] || '';
```
Or,
```js
let targetDir = '';
```
works?

Reply:
Yes, we still need that. Otherwise, it prompts for package name because `targetDir` is empty.